app/main.py

```diff
@@ -1,3 +1,16 @@
-def cache(func):
-    # Write your code here
-    pass
+from typing import Callable, Any
+
+
+def cache(func: Callable) -> Any:
+    memory = {}
+
+    def inner(*args) -> str:
+        if args in memory:
+            print("Getting from cache")
+            return memory.get(args)
+        else:
```

Comment:
There is no need in this `else` statement, code will not execute after `return`. Delete it.